bin/swagger-tools

```diff
@@ -35,9 +35,13 @@ var path = require('path');
 var pkg = require('../package.json');
 var program = require('commander');
 var request = require('superagent');
-var S = require('string');
 var YAML = require('js-yaml');
 
+String.prototype.padRight = function (length) {
```

Comment:
Instead of altering `String.prototype`, can we just make this a function and invoke it that way?